src/main/kotlin/com/jetbrains/snakecharm/lang/psi/types/SmkWildcardsType.kt

```diff
@@ -36,8 +36,7 @@ class SmkWildcardsType(private val parentDeclaration: SmkRuleOrCheckpoint) : PyT
         val resolveResult =
                 resolveToFileWildcardConstraintsKwarg(name) ?:
                 resolveToRuleWildcardConstraintsKwarg(name) ?:
-                resolveToFirstDeclaration(name) ?:
-                        return emptyList()
+                resolveToFirstDeclaration(name)
 
         return listOf(RatedResolveResult(RatedResolveResult.RATE_NORMAL, resolveResult))
```

Comment:
Not sure that it is good to return `RatedResolveResult(null)` here

Reply:
Actually, this prevents wildcard from _unresolved reference_ highlighting, instead we have an inspection that tells us that wildcard was not defined. Are there any other methods to prevent reference from being highlighted as unresolved?  Furthermore, code in example above:
```
rule foo:
  output: "{sample}.txt"
  log: "sample.txt"
```
works fine, because there are no wildcards in log section (snakemake allows this).